js/ui/save-html.js

```diff
@@ -31,15 +31,19 @@ define(
                     ,   textAlign:      "center"
                     ,   fontSize:       "inherit"
                     }
-                ,   addButton = function (title, content, fileName, popupContent) {
+                ,   addButton = function (title, href, fileName, popupContent, type) {
```

Comment:
Would suggest refactoring the `addButton` method to take an object with named props at this point.

But feel free to commit as is.